kolibri/plugins/learn/assets/src/views/HybridLearningContentCardListView.vue

```diff
@@ -70,7 +70,7 @@
       </span>
     </router-link>
     <KFixedGrid :numCols="10" class="footer">
-      <KFixedGridItem span="6" class="footer-elements">
+      <KFixedGridItem span="3" class="footer-elements">
```

Comment:
This is a bit confusing because `numCols` is 10 but the grid item spans are 3 and 10, which don't add up.

If these items are always on separate rows, we don't need the grid at all and can just use two divs.